string action #drive "fwd" OR turn the robot "left" or "right" by 90 degrees
float32 distance #instruction to drive forward by the indicated distance
